account_banking_sepa_credit_transfer/data/payment_type_sepa_sct.xml

```diff
@@ -40,6 +40,14 @@
     <field name="ir_model_id" ref="model_banking_export_sepa_wizard"/>
 </record>
 
+<record id="export_sepa_sct_001_003_03" model="payment.mode.type">
+    <field name="name">SEPA Credit Transfer pain 001.003.03 (only for Germany)</field>
```

Comment:
Change text "only for Germany" with "used in Germany", because as a standard, it can be used elsewhere.